containers/home.tsx

```diff
@@ -1,66 +1,92 @@
-import React from 'react';
-import Head from 'next/head'
-import styles from '../styles/Home.module.css'
+import React, { useState, useEffect } from 'react';
+import Head from 'next/head';
+
+// Components
+import ListCardVideo from '../components/listCardVideo/listCardVideo';
+import ArrowPrevSVG from '../components/svg/arrowPrevSVG';
+import ArrowNextSVG from '../components/svg/arrowNextSVG';
+
+// Style
+import styles from '../styles/Home.module.css';
+interface iVideo {
```

Comment:
```suggestion

interface iVideo {
```

We could break the line here just to make it crystal clear that the interface doesn't correspond to the styles.